src/plugins/subsystems/drm/drm.sh

```diff
@@ -349,9 +372,65 @@ function get_supported_mode_per_connector()
   printf '%s\n' "$modes"
 }
 
+# Creates a binary EDID file, for each connected monitor, on the current folder.
+function get_edid()
+{
+  declare -a monitors
+
+  monitors=( $(ls -d /sys/class/drm/*/edid) )
+
+  say ""
```

Comment:
I don't think you need this say, just use `\n` or something like that when you print the final result.